### PR TITLE
Make dystonse-gtfs-importer a universal tool

### PR DESCRIPTION
**Pull request by lenaschimmel**
This PR makes `import` a command within `dystonse-gtfs-tool` and adds a new `analyse` command (which does nothing yet).

If we change it that way, of course we should also rename the repository from `dystonse-gtfs-importer` to `dystonse-gtfs-tool` and adapt the Docker setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,30 @@
-# dystonse-gtfs-importer
+# dystonse-gtfs-tool
 
-This is a Rust crate that reads a static gtfs schedule file and any number of gtfs-realtime .pb or .zip files (given as command line arguments), matches the realtime data to the schedule data and writes everything into a mysql database.
+This is a Rust crate that works with a static gtfs schedules (as zip or directory), gtfs-realtime data (as .pb or .zip files) and a mysql database (setup info is specified in [dystonse-docker](https://github.com/dystonse/dystonse-docker)) to read, import or anaylse the data.
```
Comment (flauschzelle) [inline README.md]:
```suggestion
This is a Rust crate that works with static gtfs schedules (as zip or directory), gtfs-realtime data (as .pb or .zip files) and a mysql database (setup info is specified in [dystonse-docker](https://github.com/dystonse/dystonse-docker)) to read, import or anaylse the data.
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,30 @@
-# dystonse-gtfs-importer
+# dystonse-gtfs-tool
```
Comment (flauschzelle) [inline README.md]: "Tool" is a very nondescript name... the name should be indicating what this tool is for. It has more than one use case obviously, but what do all the uses have in common that makes it necessary to keep them together in one crate? Maybe they all have to access the database directly (and provide interfaces for any other part of our software that might want to use data from the database)? Then we could call it `dystonse-gtfs-db` or `dystonse-gtfs-database` or maybe even `dystonse-gtfs-data`...

File: src/analyser/mod.rs [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+            .subcommand(App::new("count"))
+        .arg(Arg::with_name("dir")
+                .index(1)
+                .value_name("DIRECTORY")
+                .required_unless("help")
+                .help("The directory which contains schedules and realtime data")
+                .long_help(
+                    "The directory that contains the schedules (located in a subdirectory named 'schedules') \
+                    and realtime data (located in a subdirectory named 'rt'). \
+                    Successfully processed files are moved to a subdirectory named 'imported'. \
```
Comment (flauschzelle) [inline src/analyser/mod.rs]: This needs to be updated, it still uses the old text (of the importer).

File: src/analyser/mod.rs [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        if rt_filenames.is_empty() {
+            return Err(Box::from(SimpleError::new("No realtime data.")));
+        }
+
+        let mut con = self.main.pool.get_conn()?;
+        let (start, end): (mysql::chrono::NaiveDateTime, mysql::chrono::NaiveDateTime) = con
+            .query_first("SELECT MIN(time_of_recording), MAX(time_of_recording) from realtime")?
+            .unwrap();
+
+        let step = Duration::hours(1);
```
Comment (flauschzelle) [inline src/analyser/mod.rs]: make this a constant or an argument to be input by the user

File: src/importer/mod.rs [hunk 5]
```diff
@@ -0,0 +1,512 @@
...
+
+use per_schedule_importer::PerScheduleImporter;
+
+const TIME_BETWEEN_DIR_SCANS: time::Duration = time::Duration::from_secs(60);
+
+pub struct Importer<'a>  {
+    main: &'a Main,
+    args: &'a ArgMatches,
+    rt_dir: Option<String>,
+    schedule_dir: Option<String>,    target_dir: Option<String>,
```
Comment (flauschzelle) [inline src/importer/mod.rs]:
```suggestion
    schedule_dir: Option<String>,    
    target_dir: Option<String>,
```

File: src/main.rs [hunk 6]
```diff
@@ -43,57 +31,10 @@ fn main() -> FnResult<()> {
...
-                .required_unless("help")
-            ).arg(Arg::with_name("rt")
-                .index(2)
-                .multiple(true)
-                .value_name("PBs")
-                .help("One or more files with real time data, as .pb or .zip")
-                .required_unless("help")
-            )
-        ).arg(Arg::with_name("verbose")
+    let matches = App::new("Dystonse GTFS Tool")
```
Comment (flauschzelle) [inline src/main.rs]: use kebap-case here (and change the name, see comments above for that)

**Review by flauschzelle — CHANGES_REQUESTED** [review]

**Comment by lenaschimmel** [comment]
I think I implemented all of your requests.

**Pull request merged**